### PR TITLE
add: Wishlist-Cart-Landing Page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Landing Page -
Screenshot 2022-02-16 at 11 45 17 AM

Wishlist Page - 
Screenshot 2022-02-16 at 11 45 26 AM

Cart Page - 
Screenshot 2022-02-16 at 11 45 41 AM

### CODE REVIEW

**Review by tarunsankhla — APPROVED** [review]
In preview on product list page on filter aside you can remove the scroll on x axis for filter doesn't look that good 

On the cart page the card heart icon was overlapping with prize added a gap to it

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <!-- horizontal light card -->
+            <div class="card-horizontal card-dark">
+                <div class="card-img-container">
+                    <img src="./src/assets/images/camera-canon.png" alt="product-image" />
```
Comment (tarunsankhla) [inline wishlist.html]: Add loading lazy

File: style.css [hunk 2]
```diff
@@ -1,13 +1,78 @@
+/* landing page */
+
+.body-landing-page,
+.dark-nav-bar,
+.dark-nav-bar input[type="search"] {
+    background-color: rgb(38, 38, 38);
```
Comment (PuneetP16) [inline style.css]: use variables instead of direct values.

File: style.css [hunk 3]
```diff
@@ -19,62 +84,184 @@
...
 }
 
-form > div > button{
+form>div>button {
     text-decoration: underline;
     align-self: flex-start;
 }
 
 .modal-active-shadow {
     box-shadow: 0 0 0 300rem rgba(0, 0, 0, 0.7);
```
Comment (PuneetP16) [inline style.css]: here too.

**Review by PuneetP16 — APPROVED** [review]
Your UI is awesome, Great Job. LGTM

**Review by tarunsankhla — APPROVED** [review]
In Preview of home page the cards are over lapping

**Comment by saurabhchirde** [comment]
@tarunsankhla It's not mobile-friendly, yet to work on the responsiveness. That's why it will overlap icons on mobile and also will show scrollbar on the x-axis of the filter section.

File: style.css (hunk 2 above)
Comment (saurabhchirde) [reply style.css]: Noted. Thank you.

File: style.css (hunk 3 above)
Comment (saurabhchirde) [reply style.css]: Here I am playing with the alpha channel, that's why didn't use variables.

File: wishlist.html (hunk 1 above)
Comment (saurabhchirde) [reply wishlist.html]: Noted. Thank you.

**Review by amarphule — APPROVED** [review]
great work, the code looks well formatted.

**Comment by sushmita12345** [comment]
Clean code. Nice UI, looks good to merge.

**Review by sushmita12345 — APPROVED** [review]

**Review by AshutoshM29 — COMMENTED** [review]
Looks Great!! can be merged

**Review by VLeads — APPROVED** [review]
Awesome UI. Code also looks clean & good

**Pull request merged**